src/stores/cdn.js

```diff
@@ -55,9 +55,15 @@ class Cdn {
   uploadObject(objectPath, filePath) {
     const metadata = this._getDefaultMetadata();
     const stream = fs.createReadStream(filePath);
-    const sanitizedObjectName = objectPath.replace(/\\/g, '/');
```

Comment:
I think we used to have some issues in S2 due to the `\` (it was considering the object name a path or something like that?). Is this sanitization not needed anymore?

Reply:
I think it is a relic from elmu days, where we did not properly slugify all uploaded file names. Now we can be sure that no path separators or other URL-unsafe characters are in the file names.